lona/html/nodes/forms/radio_button.py

```diff
@@ -0,0 +1,136 @@
+from lona.html.nodes.forms.inputs import TextInput
+from lona.html.node import Node
+
+
+class RadioButton(TextInput):
+    INPUT_ATTRIBUTE_NAME = 'checked'
+
+    ATTRIBUTES = {
+        'type': 'radio',
+        'name': 'radio',
+    }
+
+    def __init__(
+            self,
+            *args,
+            value='',
+            bubble_up=True,
+            checked=False,
+            render_value=True,
+            **kwargs,
+    ):
+
+        self.render_value = render_value
+
+        super().__init__(*args, bubble_up=bubble_up, **kwargs)
+
+        self.value = value
+        self.checked = checked
+
+    def _render_value(self, value):
+        return str(value)
+
+    # value
+    @property
+    def value(self):
+        return self._value
+
+    @value.setter
+    def value(self, new_value):
+        with self.lock:
+            if self.render_value:
+                self.attributes['value'] = self._render_value(new_value)
+
+            self._value = new_value
+
+    # name
+    @property
+    def name(self):
+        return self.attributes['name']
+
+    @name.setter
+    def name(self, new_value):
+        self.attributes['name'] = new_value
+
+    # checked
+    @property
+    def checked(self):
+        return 'checked' in self.attributes
+
+    @checked.setter
+    def checked(self, new_value):
+        if new_value:
+            self.attributes['checked'] = ''
+
+        else:
+            del self.attributes['checked']
+
+
+class RadioGroup(Node):
+    TAG_NAME = 'form'
```

Comment:
The [HTML5 specification](https://html.spec.whatwg.org/multipage/forms.html#the-form-element) explicitly does not allow nesting of `<form>`. Even if it feels like there are not many use-cases where one want's to use actual HTML forms in Lona. Why make using them extra hard? 

Is there any gain from using `form` here instead of an element like `div` that is intended for nesting?